### PR TITLE
fix #1117 : scss syntax error

### LINKED ISSUE #1117: Syntax error while compiling scss
screen shot 2017-08-05 at 2 15 40 am

### PR DESCRIPTION
**Pull request by the-dagger**
Fixes issue #1117

### CODE REVIEW

File: frontend/src/css/modules/submission.scss [hunk 1]
```diff
@@ -6,3 +6,4 @@
 
 .time {
 	position: static;
```
Comment (aayusharora) [inline frontend/src/css/modules/submission.scss]: Looks good !!

**Comment by RishabhJain2018** [comment]
@the-dagger Can you please add the fixes for urls also as shown in this [commit](https://github.com/Cloud-CV/EvalAI/commit/3fab1a8ca96f2e6a4dd4ed9a53d8ca23d91566aa#diff-14a845646ed48757b5c7798b7ccdddb6L912), so that it will be a single PR that incorporates all the fixes for  the blocked pipeline. :)

**Comment by the-dagger** [comment]
@RishabhJain2018 Thanks for the heads up, will do so 🙂

File: frontend/src/js/controllers/challengeCtrl.js [hunk 2]
```diff
@@ -909,7 +909,7 @@
             vm.remainingTime = {};
             vm.showClock = false;
             vm.showSubmissionNumbers = false;
-            parameters.url = "jobs/"+ vm.challengeId + "/phases/"+ vm.phaseId + "/remaining-submissions/";
+            parameters.url = "jobs/"+ vm.challengeId + "/phases/"+ vm.phaseId + "/remaining_submissions/";
```
Comment (deshraj) [inline frontend/src/js/controllers/challengeCtrl.js]: @the-dagger can you please remove this since this is not the part of the error that you are solving here. It would be better if this comes as a separate pull request.

Comment (the-dagger) [inline frontend/src/js/controllers/challengeCtrl.js]: @deshraj thanks for the review.
Updated this pr and opened a new pr #1120
